contact.html

```diff
@@ -0,0 +1,208 @@
+                        </li>
+                        <li class="nav-item">
+                            <a href="./about.html" class="nav-link">About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="./services.html" class="nav-link">Services</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="./blog.html" class="nav-link">Blog</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="./contact.html" class="nav-link">Contact</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </nav>
+
+<main>
+   <div class="#" style="background-color: #E2F6CD; text-align: center; padding-top: 30px; padding-bottom: 200px;">
+        <div class="row">
+            <div class="col">
+
+                <div class="container ">
+                    <div class="row" style="padding-top: 100px;" >
+                        <div class="col">
+                            <p><i>Contact Me</i></p>
+                            <h1>I'm a title. ​Click here to edit me.</h1>
+                            <p>I'm a paragraph. Click here to add your own text and edit me. I’m a great place for you to tell a story and let your
+            users know a little more about you.</p>
+                        </div>
+                    </div>
+                </div>
+
+                 <!-- <div class="container"> -->
+                    <div class="row" style="padding-top: 50px;">
+                        <div class="col-3 offset-md-3">
+                            <input class="form-control" type="text" placeholder="Name" aria-label="default input">
+                        </div>
+                        <div class="col-3 ">
+                            <input type="email" class="form-control" id="exampleFormControlInput1" placeholder="Email">
+                        </div>
+                    </div>
+                </div>
+    
+                <div class="container">
+                    <div class="row" style="padding-top: 10px;">
+                        <div class="col-md-6 offset-md-3">
+                            <input class="form-control" type="text" placeholder="Subject" aria-label="default input">
+                        </div>
+                    </div>
+                </div>
+                   
+                <div class="container">
+                    <div class="row" style="padding-top: 10px;">
+                        <div class="col-md-6 offset-md-3">
+                            <textarea class="form-control" id="exampleFormControlTextarea1" rows="5" placeholder="Message"></textarea>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="container">
```

Comment:
You don't need to repeat the `container` for each row. Just wrap your form inside one container and then just repeat `row` and `col-...` inside.